src/diffusers/pipelines/stable_diffusion/safety_checker.py

```diff
@@ -78,3 +78,29 @@ def forward(self, clip_input, images):
             )
 
         return images, has_nsfw_concepts
+
+    @torch.no_grad()
```

Comment:
Would it make sense to use `@torch.inference_mode()` instead? https://pytorch.org/docs/stable/generated/torch.inference_mode.html

Reply:
Finally a place to use it! :tada: